modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk/query_processor_test.clj

```diff
@@ -378,7 +378,7 @@
                                  :limit    1})
                 filter-clause (get-in query [:query :filter])]
             (mt/with-everything-store
-              (is (= [(str "timestamp_millis(v3_sample_dataset.reviews.rating)"
```

Comment:
Wasn't sure what `v3_sample_dataset` refers to. Is this an OK change to make?

Reply:
Bigquery has to qualify everything with the dataset name so when you do `(mt/dataset sample-dataset ...)` BigQuery ends up creating a dataset called `v3_sample_dataset`. The name is derived from the `defdataset` so if you changed the name of that from `test-dataset` to `test-database` then this would have to change too. But as I mentioned above you don't need to change the backend tests